.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+<!--
+    If you are adding a module or resources, describe what usage will be supported when this change is applied. If you are changing an existing module or resource, describe why that configuration is no longer acceptable. If you are fixing a misconfiguration, make sure to describe both the intended result and the actual (faulty) result.
+
+<!--
+    If you are adding a module or resources, describe what usage will be supported when this change is applied.
+-->
+
+<!--
+    Explain what alternative configurations you have evaluated and why the chosen configuration is preferable.
+-->
+
+<!--
+    What do reviewers need to know to review? Link pull requests or issues that are related.
+-->
+
+<!--
+    What do we need to consider when rolling out this change?
+-->
+
+Fixes #NNN <!-- This will close the issue automatically when you merge -->
+
+Completion workflow:
+
+- [ ] Module docs reflect these changes
+- [ ] All pull request comments are addressed
+- [ ] Rebase your branch against main branch
+- [ ] All checks pass with the latest changes on main branch
```

Comment:
Some of these things are already automatic checks. Others should probably be resolved through automatic checks.

Reply:
I tried to avoid those that are automatic except the final pass check which is more motivated by the workflow; I suppose it could be folded into the rebase entry. What other items are covered by automatic checks? (We could of course turn on "Require conversation resolution before merging".)